.buildkite/filebeat/filebeat-pipeline.yml

```
@@ -255,6 +256,27 @@ steps:
     if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*[Ww]indows.*/
 
     steps:
+      - label: ":windows: Filebeat: Win 2025 Unit Tests"
+        key: "windows-extended-2025"
+        command: |
+          Set-Location -Path filebeat
+          mage build unitTest
```

As far as I see mage supports running from a directory:

>   -d <string> 
>            directory to read magefiles from (default "." or "magefiles" if exists)

What's the reason for using `Set-Location -Path filebeat` instead of `mage -d filebeat build unitTest`?


```suggestion
          mage -d filebeat build unitTest
```

---

This approach was discussed and agreed during the migration, for some beats -d opt produces error, so in sake of consistency we keep it like that

---

cool, do you mind linking the GH issue or PR with the discussion?

---

I couldn't find a slack thread or a PR, since it was somewhere in the beginning of 2024. But I recall that this discussion occurred on one of our weekly meetings a year ago.